openfl-workspace/keras_nlp/src/nlp_dataloader.py

```diff
@@ -91,7 +95,10 @@ def get_valid_data_size(self):
         return self.X_valid[0].shape[0]
 
     @staticmethod
-    def _batch_generator(X1, X2, y, idxs, batch_size, num_batches):
+    def _batch_generator(X1: np.ndarray, X2: np.ndarray,
+                         y: np.ndarray, idxs: np.ndarray,
+                         batch_size: Optional[int],
+                         num_batches: int) -> Iterable[List[np.ndarray]]:
```

Comment:
Should be:
```suggestion
                         num_batches: int) -> Generator[type1, type2, type3]:
```

Reply:
I think that using `Iterable` instead of `Generator` still fine.